### ISSUE TITLE
Add rule for "technical controls"

### ISSUE THREAD

**Issue by junckritter** [kiwicom/contessa#40]
Add rule defining SQL template for checking if count or sum in one column in last day is within the range typical for last week/month.
Instance of the rule then needs e.g. only name of the column, time interval and percentage.

**Comment by maroshmka** [comment]
Should Contessa be a library for this ? You can detect anomalies in other services (e.g. Datadog), or even in simple dashboard build upon the check results. 

I think, Contessa can check that if the sum(column_A) is bigger than 20 in specific interval and report it to auditlog database. What you're gonna do with it its up to you.

We may include some basic anomaly detection in the future, but I would start with the SUM rule.

**Comment by junckritter** [comment]
I agree that the rules in Contessa should be "general", i.e. not specific and this is rather advanced. But let's see if we can define some "simpler" version with basic template. 
The problem with your example is, that user has to somehow find the value _20_. Maybe it would be enough, if the rules allows to define also "reference" query (baseline in bigger timeframe) and then compare that with the actual query. 

It can be seen as step towards set of automatic advanced rules.
